src/widget/wsearchlineedit.cpp

```diff
@@ -69,7 +69,9 @@ WSearchLineEdit::WSearchLineEdit(QWidget* pParent)
     DEBUG_ASSERT(!kEmptySearch.isNull());
 
     setAcceptDrops(false);
-    setPlaceholderText(tr("Search...", "noun"));
+
+    //: This should be a noun.
```

Comment:
I don't think this really explains the context a translator would need to understand how this string is used.
```suggestion
    //: Shown in the library search bar when it is empty.
```